reference-architecture/aws-ansible/ose-on-aws.py

```diff
@@ -72,7 +72,7 @@
 @click.option('--s3-bucket-name', help='Bucket name for S3 for registry')
 @click.option('--github-client-id', help='GitHub OAuth ClientID')
 @click.option('--github-client-secret', help='GitHub OAuth Client Secret')
-@click.option('--github-organization', prompt=True, multiple=True, help='GitHub Organization')
+@click.option('--github-organization', prompt=True, help='GitHub Organization')
```

Comment:
@cooktheryan ah, don't mistake it for `nargs` where an option can take multiple arguments, `multiple=True` simply allows for the option to be specified multiple times and the variable will return all values that were specified.